rd-net/CrossTest/TestRunner.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Linq;
+
+namespace Test.RdCross
+{
+    public class TestRunner
+    {
+        public static void Main(string[] args)
+        {
+            if (args.Length < 1)
+            {
+                throw new ArgumentException($"Wrong number of arguments:${args.Length}");
```

Comment:
Throw better diagnostics. What is expected number of arguments and what is the argument. Example would be perfect.